.devcontainer/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Image for a NYU Lab development environment
-FROM rofrano/nyu-devops-base:su23
+FROM rofrano/pipeline-selenium:latest
```

Comment:
Rebase off mainline? I just merged in the other story you made where included this change